docs/service-loadbalancer.md

```diff
@@ -0,0 +1,288 @@
+	- [Create a Service of type LoadBalancer](#create-a-service-of-type-loadbalancer)
+	- [Validate Service external IP](#validate-service-external-ip)
+  - [Limitations](#limitations)
+- [Using MetalLB with Antrea](#using-metalb-with-antrea)
+
+In Kubernetes, the implementation of Services of type LoadBalancer usually
+requires an external load balancer. On cloud platforms (including public clouds
+and platforms like NSX-T) which support load balancers, Services of type
+LoadBalancer can be implemented by the Kubernetes Cloud Provider which
+configures the cloud load balancers for the Services. However, the load
+balancer support is not available on all platforms, or in some cases it is
+complex or has extra cost to deploy external load balancers. This document
+describes two options of supporting Services of type LoadBalancer with Antrea,
+without an external load balancer:
+
+1. Using Antrea's built-in external IP management for Services of type
+LoadBalancer
+2. Leveraging [MetalLB](https://metallb.universe.tf)
+
+## Service external IP management by Antrea
+
+Antrea starts to support external IP management for Services of type
+LoadBalancer since version 1.5, which can work together with `AntreaProxy` or
+`kube-proxy` to implement Services of type LoadBalancer, without requiring an
+external load balancer. With the external IP management feature, Antrea can
+allocate an external IP for a Service of type LoadBalancer from an
+[ExternalIPPool](egress.md#the-externalippool-resource), and select a Node
+based on the ExternalIPPool's NodeSelector to host the external IP. Antrea
+configures the Service externalIP to the selected Node, and thus the Service
+requests to the external IP will get to the Node, and they are then handled by
+`AntreaProxy` or `kube-proxy` on the Node and distributed to the Service's
+Endpoints. Antrea also implements a Node failover mechanism for Service
+external IPs. When Antrea detects a Node hosting an external IP is down, it
+will move the external IP to another available Node of the ExternalIPPool.
+
+### Configuration
+
+#### Enable Service external IP management feature of `antrea-agent`
+
+At this moment, external IP management for Services is an alpha feature of
+Antrea. The the `ServiceExternalIP` feature gate of `antrea-agent` must be
+enabled for the feature to work. You can enable the `ServiceExternalIP`
+featue gate in the `antrea-config` ConfigMap in the Antrea deployment YAML:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: antrea-config-dcfb6k2hkm
+  namespace: kube-system
+data:
+  antrea-agent.conf: |
+    featureGates:
+...
+      ServiceExternalIP: true
+...
+```
+
+The feature works with both `AntreaProxy` and `kube-proxy`, including the
+following configurations:
+
+- `AntreaProxy` without `proxyAll` enabled - this is `antrea-agent`'s default
+configuration, configuration, in which `kube-proxy` serves the request traffic
```

Comment:
either "default configuration, configuration in which" or "default configuration, in which"